packages/php/email-editor/src/Engine/PersonalizationTags/class-personalization-tags-registry.php

```diff
@@ -39,10 +58,27 @@ public function initialize(): void {
 	 */
 	public function register( Personalization_Tag $tag ): void {
 		if ( isset( $this->tags[ $tag->get_token() ] ) ) {
+			$this->logger->warning(
+				'Personalization tag already registered',
+				array(
+					'token'    => $tag->get_token(),
+					'name'     => $tag->get_name(),
+					'category' => $tag->get_category(),
+				)
+			);
 			return;
 		}
 
 		$this->tags[ $tag->get_token() ] = $tag;
+
+		$this->logger->debug(
```

Comment:
@yuliyan hey 👋 This debug log is bloating my debug file :) Can we do something about it? Is there a way to disable the email editor and personalization registry loggings?

![image](https://github.com/user-attachments/assets/20143cb5-9357-4a83-8948-d7d8a6cb43dd)

Reply:
Guess that's happening because I enabled the new React based email editor to see the new emails I added is compatible with the new UI. I'll try to disable it again.

Reply:
@tpaksu, Thanks for the update! Just to confirm, does everything work as expected with the block email editor disabled? Let me know if there's anything else I should check.

Reply:
@yuliyan yes everything works as usual. The logs stopped when I disabled the new block email editor. I added an edit to the previous comment, but guess you missed it 🙂